.github/workflows/integration.yml

```diff
@@ -0,0 +1,111 @@
+name: Integration Tests
+
+concurrency: IntegrationTest
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
```

Comment:
Can we set it so that this action doesn't get triggered by simple documentation changes to `*.md`? Similar note for other actions.